src/index.ts

```diff
@@ -1,81 +1,53 @@
 import * as WebSocket from 'ws';
-import KaiEventEmitter from './KaiEventEmitter';
-
-let ws: WebSocket;
-
-ws.on('open', () => {
-	// TODO:
-	// Test compartibility
-	// Send capabilities
-});
-
-ws.on('message', (data) => {
-	var input = JSON.parse(data.toString());
-	KaiEvents.emit('data', input);
-
-	if(input.success != true) {
-		KaiEvents.emit('error', {
-			errorCode: input.errorCode,
-			error: input.error,
-			message: input.message
-		});
-		return;
-	}
-
-	switch(input.type) {
-		case "gestureData":
-			KaiEvents.emit('gestureData', input.gesture);
-			break;
-		// TODO other types of data
+import { stringify } from 'querystring';
+import * as sdk from './sdk';
+import { print } from 'util';
+import { type } from 'os';
+import { resolve } from 'path';
+import { rejects } from 'assert';
+import * as Promise from 'bluebird';
+let ws:WebSocket;
+var isConnected:boolean = false
+var isAuthenticated:boolean = false
+
+
+connect('123','qwerty')
```

Comment:
Remove this test connect code. Add a test.ts file to test out the library, avoid adding these in the actual code

Reply:
fixed